metrics/config_test.go

```diff
@@ -93,13 +93,24 @@ var (
 		name: "invalidReportingPeriod",
 		ops: ExporterOptions{
 			ConfigMap: map[string]string{
-				"metrics.backend-destination":      "prometheus",
+				"metrics.backend-destination":      "opencensus",
 				"metrics.reporting-period-seconds": "test",
 			},
 			Domain:    servingDomain,
 			Component: testComponent,
 		},
 		expectedErr: "invalid metrics.reporting-period-seconds value \"test\"",
+	}, {
+		name: "invalidOpenCensusSecuritySetting",
```

Comment:
validOpenCensusSecuritySetting test?

Reply:
Good point, done.

(BTW, I find it strange that this table test doesn't simply have a *err where empty means you expect success. But that's for a different PR.)